### PR TITLE
Unofficial transportdocument and to

### PR DESCRIPTION
**Pull request by pimg**
Added base project structure and implemented two unofficial endpoints. No Integration tests have been setup (yet) for these unofficial endpoints.

### CODE REVIEW

File: ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/service/TransactionService.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  public Optional<UUID> createLocalTransaction(
+      String documentHash, EndorsementChainTransaction transactionRequest) {
+
+    TransportDocument transportDocumentOptional =
+        transportDocumentRepository
+            .findById(documentHash)
+            .orElseThrow(
+                () -> ConcreteRequestErrorMessageException.notFound("TransportDocument not found"));
+
+    return Optional.ofNullable(transportDocumentOptional)
```
Comment (nt-as) [inline ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/service/TransactionService.java:38]: `transportDocumentOptional` cannot be null here (due to `.orElseThrow` above)

File: ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/mappers/TransactionMapper.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import org.dcsa.endorsementchain.transferobjects.EndorsementChainTransaction;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(componentModel = "spring")
+public interface TransactionMapper {
+
+  @Mapping(expression = "java(System.currentTimeMillis())", target = "timestamp")
+  @Mapping(source = "platformName", target = "platformHost")
+  Transaction EndorsementChainTransactionToTransaction(EndorsementChainTransaction endorsementChainTransaction, String platformName);
```
Comment (nt-as) [inline ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/mappers/TransactionMapper.java:13]: `endorsementChainTransactionToTransaction` to follow java naming conventions?

File: ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/service/TransactionService.java (hunk 1 above)
Comment (pimg) [reply ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/service/TransactionService.java]: fixed

File: ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/mappers/TransactionMapper.java (hunk 2 above)
Comment (pimg) [reply ec-service/src/main/java/org/dcsa/endorsementchain/unofficial/mappers/TransactionMapper.java]: fixed

**Review by nt-as — APPROVED** [review]

**Pull request merged**